infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf

```diff
@@ -1,41 +1,28 @@
 /*
 Copyright 2022 The Kubernetes Authors.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-locals {
-  project_id = "k8s-infra-oci-proxy-prod"
-  domain     = "registry.k8s.io"
-  image      = "us.gcr.io/k8s-artifacts-prod/infra-tools/archeio:${var.tag}"
-
-}
-
-data "google_billing_account" "account" {
-  billing_account = "018801-93540E-22A20E"
-}
-
 data "google_organization" "org" {
   domain = "kubernetes.io"
 }
 
 resource "google_project" "project" {
-  name            = local.project_id
-  project_id      = local.project_id
+  name            = var.project_id
+  project_id      = var.project_id
   org_id          = data.google_organization.org.org_id
-  billing_account = data.google_billing_account.account.id
+  billing_account = "018801-93540E-22A20E"
```

Comment:
what's wrong with the data object ?

Reply:
It requires access to the billing account(ie, being member of `k8s-infra-oci-proxy-admins@kubernetes.io` alone will fail) and you aren't really doing anything special with it. The same value is passed to the `google_project` resource.